week-1-practise/level-1/8-function-output-1.js

```diff
@@ -6,7 +6,7 @@ var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? it returns a random number and multiplies it by 10.
```

Comment:
Minor / nitpick: the program multiplies the result of Math.random by 10 before returning. So it might be a bit clearer to state:

> it returns a (floating-point) random number between 0 and 10